javascript/react-verbs/docs/react-verbs.api.md

```diff
@@ -317,7 +328,7 @@ export type StepChangeFunction = (step: Step) => void;
 // Warning: (ae-missing-release-tag) "StepComponentProps" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export interface StepComponentProps extends StepDependent {
+export interface StepComponentProps<T extends object = any> extends StepDependent<T> {
```

Comment:
Shouldn't this extend `unknown` by default?

Reply:
Yep, unknown is reasonable